source/dub/internal/utils.d

```diff
@@ -427,43 +427,32 @@ string getModuleNameFromContent(string content) {
 	static Regex!char comments_pattern, module_pattern;
 
 	if (!regex_initialized) {
-		comments_pattern = regex(`(/\*([^*]|[\r\n]|(\*+([^*/]|[\r\n])))*\*+/)|(//.*)`, "g");
+		comments_pattern = regex(`//[^\r\n]*\r?\n?|/\*.*\*/|/\+.*\+/`, "g");
 		module_pattern = regex(`module\s+([\w\.]+)\s*;`, "g");
 		regex_initialized = true;
 	}
 
 	content = replaceAll(content, comments_pattern, "");
 	auto result = matchFirst(content, module_pattern);
 
-	string moduleName;
-	if(!result.empty) moduleName = result.front;
+	if (!result.empty) return result[1];
 
-	if (moduleName.length >= 7) moduleName = moduleName[7..$-1];
-
-	return moduleName;
+	return null;
```

Comment:
I could have used "" here, because the user code checks for `.length == 0` anyway, but I'm kind of used to using `null`. But in general, I think that using an empty string to denote the absence of a `module` statement should be fine here, as there are no ambiguities and `Nullable` would mainly just complicate the usage.